mitosheet/src/components/taskpanes/Graph/GraphStyleTab.tsx

```diff
@@ -186,6 +194,51 @@ function GraphStyleTab(props: {
                         }}
                     />
                 }
+                <Row justify='space-between' align='center'>
+                    <Col>
+                        <p>
+                            Type
```

Comment:
I think the name "type" is kinda confusing. Can we name it Axis Transform (and then maybe change default to None) or something?

Or I'm misunderstanding what this is, and probably default doesn't mean None. But I think calling this transform instead of type might be helpful, idk...